client/src/datafiles/layouts/projects/ProjectPipelineSelectLayout.tsx

```diff
@@ -11,6 +11,9 @@ export const ProjectPipelineSelectLayout: React.FC = () => {
     (e) => e.value.dois && e.value.dois.length > 0
   );
 
+  const projectType = data.baseProject?.value?.projectType; //getting selected project type value
+  //console.log(projectType)
```

Comment:
```suggestion
```
We want to make sure we don't include any commented lines that aren't informative/necessary.